### ISSUE TITLE
How to train the pretrained model?

### ISSUE THREAD

**Issue by KenkunLiu** [hustvl/GNeuVox#3]
Thanks for the great work~ I wonder how to train the pretrained model?

**Comment by KenkunLiu** [comment]
And what subjects are used to train the provided pretrained model?

**Comment by taoranyi** [comment]
Thank you for your interest. During pretraining, we randomly select rays in all subjects. The rays from different subjects are corresponded with different individual voxels V^i and embedding Z. However, they all go through the same pose deformation network, radiance fields, and other parts.
For model [h36m_pre](https://drive.google.com/drive/folders/1jHLttKaRvrEvGHp-yYXQH7aE5HEFLA9i?usp=share_link), we pretrained it on 7 subjects (S1, S5, S6, S7, S8, S9, and S11). As for the given model [zju_pre](https://drive.google.com/drive/folders/1jHLttKaRvrEvGHp-yYXQH7aE5HEFLA9i?usp=share_link), for convenience, we did not use leave-one-out method, but directly pretrained it on 8 subjects (313, 377, 386, 387, 390, 392, 393, 394).
